### DISCUSSION TITLE
is product details page can customize in swell?

### DISCUSSION THREAD

**Discussion by RushiNCode93** [swellstores/community#144] [Help] [answered]
hello,
can we customize the product details page like the below screenshot?

![image](https://user-images.githubusercontent.com/97941602/219300976-b37811de-50ad-4603-ae24-6aacb5ba8bc0.png)

**Comment by ericingram** [comment, marked as answer]
That shouldn't be a problem for a developer

**Reply by logeshswell** [reply]
Yes @RushiNCode93, why not?

You can customize your storefront as per your need.
